volatility3/framework/plugins/windows/modules.py

```diff
@@ -138,16 +138,22 @@ def get_kernel_space_start(cls, context, module_name: str) -> int:
         """
         module = context.modules[module_name]
 
+        # default is used if/when MmSystemRangeStart is paged out
         if symbols.symbol_table_is_64bit(context, module.symbol_table_name):
             object_type = "unsigned long long"
+            default_start = 0xFFFF800000000000
         else:
             object_type = "unsigned long"
+            default_start = 0x80000000
 
         range_start_offset = module.get_symbol("MmSystemRangeStart").address
 
-        kernel_space_start = module.object(
-            object_type=object_type, offset=range_start_offset
-        )
+        try:
+            kernel_space_start = module.object(
+                object_type=object_type, offset=range_start_offset
+            )
+        except exceptions.InvalidAddressException:
+            kernel_space_start = default_start
```

Comment:
Could you add a debug log here to say that the actual address couldn't be found and the default is being used, please?

Reply:
done